pkg/controller/import-controller.go

```diff
@@ -43,8 +43,8 @@ type ImportController struct {
 }
 
 type importPodEnvVar struct {
-	ep, secretName, source, contentType, imageSize, certConfigMap string
-	insecureTLS                                                   bool
+	ep, secretName, source, contentType, imageSize, certConfigMap, volumeMode string
```

Comment:
I don't think we need this, we can check in the importer pod for the existence of the block device file, and say based on that if we are in block mode or not. That is one less environment variable we need to set.

Reply:
How do you check in the importer pod that this is going to be block device?

Reply:
You check if the device exists by checking if the block device file exists. If not, its not volumeMode block.